common/spec/specfiles.go

```diff
@@ -39,6 +38,23 @@ func CreateSpecFromFile(specFilePath string, specVars map[string]string) (spec *
 	return
 }
 
+func CreateSpecFromVariables(buildName string, buildNumber string) (*SpecFiles, error) {
```

Comment:
- Where is this new API used?
- CreateSpecFromVariables --> CreateSpecFromBuildNameAndNumber
- (buildName string, buildNumber string) --> (buildName, buildNumber string)

Reply:
it will be used in jfrog-cli, in the RB creation command (the motivation is to make the --spec flag optional)